network-protection/network-protection-api/src/main/java/com/duckduckgo/networkprotection/api/NetworkProtectionAccessState.kt

```diff
@@ -25,13 +25,25 @@ interface NetworkProtectionAccessState {
      * Returns the state of the NetP
      * The caller DOES NOT need to specify the dispatcher when calling this method
      */
-    suspend fun getState(): NetPAccessState
+    suspend fun getLegacyState(): LegacyNetPAccessState
 
     /**
      * Returns a flow of the states of the NetP access
      * The caller DOES NOT need to specify the dispatcher when calling this method
      */
-    suspend fun getStateFlow(): Flow<NetPAccessState>
+    suspend fun getLegacyStateFlow(): Flow<LegacyNetPAccessState>
+
+    /**
+     * Returns the visibility state of the NetP
+     * The caller DOES NOT need to specify the dispatcher when calling this method
+     */
+    suspend fun getState(): NetPVisibilityState
```

Comment:
this change needs to be documented and be known to others. follow the API change process

Reply:
I've removed this and moved the state internally. See: [`82174dd` (#5394)](https://github.com/duckduckgo/Android/pull/5394/commits/82174dd47ee45383775988a0456a2a34aecb16d2)